File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cs
public class Solution {
    public string MergeAlternately(string word1, string word2) {
        StringBuilder s=new StringBuilder();
        int i=0;
        int j=0;
        while(i<word1.Length || j<word2.Length){
            if(i<word1.Length){
                s.Append(word1[i]);
                 i++;
            }
            if(j<word2.Length){
            s.Append(word2[j]);
           
            j++;
            }
        }
        return s.ToString();
    }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        res=[]
        left=0
        right=0
        while left<len(word1) or right<len(word2):
            if left<len(word1):
                res.append(word1[left])
                left+=1
            if right<len(word2):
                res.append(word2[right])
                right+=1
        return "".join(res)